src/preferences/dialog/dlgprefdeck.cpp

```diff
@@ -465,6 +464,16 @@ void DlgPrefDeck::slotUpdate() {
     index = ComboBoxCueMode->findData(static_cast<int>(cueMode));
     ComboBoxCueMode->setCurrentIndex(index);
 
+    const EngineSync::SyncLockAlgorithm syncLockAlgorithm =
+            static_cast<EngineSync::SyncLockAlgorithm>(m_pConfig->getValue<int>(
+                    ConfigKey(BPM_CONFIG_GROUP, SYNC_LOCK_ALGORITHM_CONFIG_KEY),
+                    EngineSync::SyncLockAlgorithm::PREFER_SOFT_LEADER));
```

Comment:
Seeing this snippet while starting the review a couple weeks ago was the entire reason for why I implemented #11883 :sweat_smile: 
Please update this and the snippet below when merging to `main`.